pkg/scheduler/apis/config/types.go

```diff
@@ -85,6 +85,7 @@ type KubeSchedulerConfiguration struct {
 	// then scheduler stops finding further feasible nodes once it finds 150 feasible ones.
 	// When the value is 0, default percentage (5%--50% based on the size of the cluster) of the
 	// nodes will be scored.
+	// DEPRECATED: Please use PercentageOfNodesToScore which defined in KubeSchedulerProfile
```

Comment:
```suggestion
	// DEPRECATED: Please use scheduler profiles to configure this instead.
```

Reply:
Also, do we have a plan for eventually removing this from KubeSchedulerConfiguration?

Reply:
>yeah, we could keep both in Beta, and remove the global one in GA.

https://github.com/kubernetes/kubernetes/issues/93270#issuecomment-661884459